Sources/LiveKit/Types/VideoParameters.swift

```diff
@@ -38,6 +38,38 @@ public struct VideoParameters {
     }
 }
 
+// MARK: - Computation
+
+internal extension VideoParameters {
+
+    func defaultScreenShareSimulcastLayers() -> [VideoParameters] {
+
+        struct Layer {
+            let scaleDownBy: Double
+            let fps: Int
+        }
+
+        let layers = [Layer(scaleDownBy: 2, fps: 3)]
```

Comment:
how does it work with a low fps like 3? is it sufficient?

Reply:
Pretty low fps but it does work haha, this is same with JS SDK you can try it.